recipes/repo.rb

```diff
@@ -386,7 +386,7 @@
   source 'pulp/Dockerfile'
 end
 
-if node['ros_buildfarm']['repo']['enable_pulp_services']
+if node['ros_buildfarm']['rpm_repos']
```

Comment:
Unlike Python where empty collections are false-y all objects in Ruby are truth-y except false and nil.

```suggestion
if not node['ros_buildfarm']['rpm_repos'].empty?
```

Reply:
Thanks, I expanded on this in 457f131.